client/post-editor/editor-drawer/index.jsx

```diff
@@ -296,26 +304,26 @@ const EditorDrawer = React.createClass( {
 
 		return (
 			<div className="editor-drawer">
-				{ site && ! this.currentPostTypeSupportsAll() && (
+				{ site && ! POST_TYPE_SUPPORTS[ type ] && (
```

Comment:
>I think it's a bit clearer if you called a function like `! supportsAllFeatures( type )` rather than depending on the structure of fallback object.

`supportsAllFeatures` wouldn't be necessarily accurate (as the absence of a key is intentional in indicating lack of support), but I like the premise of the idea in improving readability. Added `hasHardCodedPostTypeSupports` in rebased 95f6d224bd3a2a5e0687c6806dc38c9f955197ed.